src/applications/vaos/tests/covid-19-vaccine/components/ConfirmationPageV2.unit.spec.jsx

```diff
@@ -101,11 +100,26 @@ describe('VAOS vaccine flow <ConfirmationPageV2>', () => {
         /We’ve scheduled and confirmed your appointment./i,
       ),
     ).to.be.ok;
-    userEvent.click(screen.getByText(/Review your appointments/i));
```

Comment:
This test used to confirm what the browser does on link click with the internals of react-router. Because react-testing-library doesn't support accessing the shadow DOM, the `userEvent.click()` can't be called.